docs/source/setup/single-machine.rst

```diff
@@ -67,6 +67,6 @@ You can specify these functions in any of the following ways:
 Use the Distributed Scheduler
 -----------------------------
 
-The newer dask.distributed scheduler also works well on a single machine and
+The newer ``dask.distributed`` scheduler also works well on a single machine and
```

Comment:
Maybe just "The newer distributed scheduler"

Reply:
I think that I find switching to code highlighting like this a bit distracting.  I think that we need to find a nice way to talk about the distributed scheduler without using the module name.  

Could also be "Dask's newer distributed scheduler also works ..."

Reply:
This way seems to be better than using the the highlighted module name. I'll rectify this change asap